config/sync/jsonapi_extras.jsonapi_resource_config.comment--comment.yml

```diff
@@ -0,0 +1,135 @@
+uuid: bfaa07af-27ff-48ba-82d0-5bc1cd2ec7d6
+langcode: en
+status: true
+dependencies: {  }
+id: comment--comment
+disabled: false
+path: comments
+resourceType: comments
+resourceFields:
+  cid:
+    fieldName: cid
+    publicName: internalId
+    enhancer:
+      id: ''
+    disabled: false
+  uuid:
+    disabled: true
+    fieldName: uuid
+    publicName: uuid
+    enhancer:
+      id: ''
+  langcode:
+    disabled: true
+    fieldName: langcode
+    publicName: langcode
+    enhancer:
+      id: ''
+  comment_type:
+    fieldName: comment_type
+    publicName: commentType
+    enhancer:
+      id: ''
+    disabled: false
+  status:
+    fieldName: status
+    publicName: isPublished
+    enhancer:
+      id: ''
+    disabled: false
+  pid:
+    fieldName: pid
+    publicName: pid
+    enhancer:
+      id: ''
+    disabled: false
+  entity_id:
+    fieldName: entity_id
+    publicName: referencedInternalId
+    enhancer:
+      id: ''
+    disabled: false
+  subject:
+    fieldName: subject
+    publicName: subject
+    enhancer:
+      id: ''
+    disabled: false
+  uid:
+    fieldName: uid
+    publicName: author
```

Comment:
I think that it makes sense to have the fields represent the actual relationship. Ownership of a comment (or a piece of content) is not necessarily the same as authorship. If I pay you to write an article, I am the owner and you are the author.

That being said, I don't have any feelings about this. I'll make the change to owner.